src/constants.ts

```diff
@@ -0,0 +1,59 @@
+export const Constants = {
```

Comment:
follow uniform coding conventions in your entire application. If other constant variables are uppercase, keep this one Uppercase two. 
Variable name should also be specific like `STATIC_ASSETS_PATH`